docs/search-core.segment.pagenumber.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@yext/search-core](./search-core.md) &gt; [Segment](./search-core.segment.md) &gt; [pageNumber](./search-core.segment.pagenumber.md)
+
+## Segment.pageNumber property
+
+The page number of a segment from a pdf

```

Comment:
I'd prefer to say something like "The page number of the document that the segment came from. Only populated when document is a PDF file."
If you end up changing this, make sure to update other places as well